app/controllers/static_controller.rb

```diff
@@ -7,11 +7,13 @@ class StaticController < ApplicationController
 
   # Reference: https://github.com/froala/editor-ruby-sdk-example
   # https://www.froala.com/wysiwyg-editor/docs/sdks/ruby/image-server-upload
-
+ 
   def show
     if params[:id]
+      authenticate_user! if params[:id] == 1
```

Comment:
I believe on prod this id is 5, as from admin portal, the URL to this custom page is https://pomonastudents.org/pages/id/5